pyaedt/modules/solutions.py

```diff
@@ -1913,14 +1910,12 @@ def polar_plot_3d(
             Full path for the image file. The default is ``None``, in which case a file is not exported.
         show : bool, optional
             Whether to show the plot. The default is ``True``.
-            If ``False``, the Matplotlib instance of the plot is shown.
+            If ``False``, the Matplotlib instance of the plot is not shown.
```

Comment:
```suggestion
            If ``False``, the ``plt.Figure`` instance of the plot is not shown.
```
Just semantics, but it is the figure rendering that in changed by `show`

Reply:
Thanks for the remark ! I'll handle this comment depending on your answer to my previous question.

Reply:
@SMoraisAnsys : It wasn't clear to me what the ``pyplot`` interface was doing. Now it's clear. It is the Figure instance that is rendered, and ``pyplot`` is just along for the ride to do the bookkeeping.